src/map/conquest_system.cpp

```diff
@@ -105,7 +105,7 @@ namespace conquest
         influences[nation] += lost;
 
         Sql_Query(SqlHandle, "UPDATE conquest_system SET sandoria_influence = %d, bastok_influence = %d, "
-            "windurst_influence = %d, beastmen_influence = %d WHERE region_id = %d;", influences[0], influences[1], influences[2], influences[3], region);
+            "windurst_influence = %d, beastmen_influence = %d WHERE region_id = %d;", influences[0], influences[1], influences[2], influences[3], static_cast<uint8>(region));
```

Comment:
Since its unsigned that should be `%u` instead of `%d`.

Also this only needs cast because of a compiler bug. The first toolset that shipped with VS2019 didn't have this problem nor did any past ones, this all started with updates to visual studio. Dunno if anyone reported it to MS yet.. We should still make it explicit anyway but I think thats why nobody made this change yet. ¯\\\_(ツ)_/¯

Reply:
will there be any issues if this is changed for ppl that compile on linux?

Reply:
doubt it, prolly be just fine. travis is stuck atm or I'd point to what it says - travis builds linux for us while appveyor builds windows.